source/Mlos.Python/mlos/Spaces/HypergridAdapters/unit_tests/TestContinuousToPolynomialBasisHypergridAdapter.py

```diff
@@ -178,3 +178,42 @@ def _test_polynomial_feature_values_are_as_expected(adapter, projected_df):
                 print('expected: ', expected_values)
                 print('observed: ', observed_values)
             assert sum_diffs < epsilon
+
+    @pytest.mark.parametrize("degree", [2, 3, 5])
+    @pytest.mark.parametrize("include_bias", [True, False])
+    @pytest.mark.parametrize("interaction_only", [True, False])
+    @pytest.mark.parametrize("drop", ['first', None])
+    @pytest.mark.parametrize("merge_all_categorical_dimensions", [True, False])
+    def test_stacking_polynomial_feature_on_one_hot_encoding_parameterized(self,
+                                                                           degree,

```

Comment:
Could you please insert a line break before self, and reduce the indentation of all arguments?

Reply:
That was the formatting I started with, but pylint complained about it.  Maybe a pylint issue on my pyCharm instance, so I'll change it to the recommended formatting and follow up with you about my pylint messages if pylint blocks merge.